src/node.cc

```diff
@@ -2736,6 +2736,9 @@ void SetupProcessObject(Environment* env,
   Local<Object> release = Object::New(env->isolate());
   READONLY_PROPERTY(process, "release", release);
   READONLY_PROPERTY(release, "name", OneByteString(env->isolate(), "node"));
+  READONLY_PROPERTY(release,
```

Comment:
this PR is against `v4.x`, that's how; I'm open to suggestions on doc changes that could go in to `master` if any